backend/models/userSchema.ts

```diff
@@ -0,0 +1,18 @@
+import * as mongoose from 'mongoose';
+
+const UserSchema = new mongoose.Schema(
+  {
+    isAdmin: Boolean,
+    firstName: String,
+    lastName: String,
```

Comment:
I know we were still undecided about saving these separately or together on Monday, but I think we are okay to change this to have a single field `name` which will encapsulate both.